packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -3,130 +3,184 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-panel-background-color: Specifies the background color of the component.
+ * @prop --calcite-panel-border-color: Specifies the border color of the component.
+ * @prop --calcite-panel-description-color: Specifies the color of the component's description.
+ * @prop --calcite-panel-fab-z-index: Specifies the component fab's z-index.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
  * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
```

Comment:
are there requests/user stories for custom control of padding? I'm wondering if this token is necessary.

Reply:
This var is already there so if we want we can deprecate it.

Reply:
- [ ] deprecate old calcite-panel-footer-padding variable and replace with a "space" token